tests/wiseagents/test_yaml_serializtion.py

```diff
@@ -128,3 +129,12 @@ def test_using_deserialized_agent(monkeypatch):
         assert deserialized_agent.vector_db.embedding_model_name == "all-MiniLM-L6-v2"
     finally:
         deserialized_agent.stop_agent()
+
+def test_serialize_assistant():
+    try:
+        assistant = AssistantAgent(name="Assistant1", description="This is a test assistant", transport=DummyTransport(), destination_agent_name="")
+        serialized_assistant = yaml.dump(assistant)
+        logging.info(serialized_assistant)
+        deserialized_agent = yaml.load(serialized_assistant, Loader=yaml.Loader)
+    finally:
+        assistant.stop_agent()
```

Comment:
Just to check, is this needed? I don't see the agent being started.

Reply:
Ah, I initially thought it was only being loaded from YAML and there the constructor doesn't get called but I see the object creation now.